### PR TITLE
added mobile number to registration

### PR DESCRIPTION
**Pull request by codiebeulaine**
@saeedmb8 ready for review

### CODE REVIEW

**Comment by peterp** [comment]
@codiebeulaine I think overall the forms need some love, but the errors are now here:

screenshot 2016-04-20 15 05 00

**Comment by codiebeulaine** [comment]
@peterp 👍

**Comment by codiebeulaine** [comment]
@saeedmb8 @miltontony ready for review

File: freebasics/forms.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from django import forms
 from django.contrib.auth.models import User
 from django.utils.translation import ugettext_lazy as _
+from phonenumber_field.formfields import PhoneNumberField
 
 
 class RegistrationForm(forms.Form):
```
Comment (miltontony) [inline freebasics/forms.py]: This needs to inherit from `molo.profiles` similar to what we did in [molo.gem](https://github.com/praekelt/molo-gem/blob/develop/gem/forms.py#L7-L12)

File: freebasics/templates/profiles/register.html [hunk 2]
```diff
@@ -8,13 +8,29 @@ <h6 class="block-heading fb-accent-1">{% trans "Join" %}</h6>
...
+             <p>{% trans "(e.g. +260979000001)" %}</p>
+             {{ form.mobile_number }}
+             {{ form.mobile_number.errors}}
+             <br>
+          {% endif %}
+
+          {{ form.terms_and_conditions }}
+          <span>{% trans "I accept the Terms and Conditions" %}</span>
+          {% if form.terms_and_conditions.errors %}
+            <p class="error">{% trans "Please accept the T&Cs in order to complete the registration" %}</p>
```
Comment (miltontony) [inline freebasics/templates/profiles/register.html]: T&C's needs to be `T&amp;Cs`

**Comment by codiebeulaine** [comment]
@miltontony @saeedmb8 ready for review

**Comment by smarzban** [comment]
👍  🍰

**Pull request merged**